cassandra-four-zero-bridge/src/main/java/org/apache/cassandra/spark/reader/SSTableCache.java

```diff
@@ -108,6 +111,27 @@ public BloomFilter bloomFilter(@NotNull SSTable ssTable, Descriptor descriptor)
         return get(filter, ssTable, () -> ReaderUtils.readFilter(ssTable, descriptor.version.hasOldBfFormat()));
     }
 
+    public CompressionMetadata compressionMetaData(@NotNull SSTable ssTable, boolean hasMaxCompressedLength) throws IOException
+    {
+        if (!"true".equalsIgnoreCase(System.getProperty("sbr.cache.compression.enabled", "true")))
```

Comment:
How about naming the property `sbr.cache.compressionInfo.enabled`? matching the name of the component (compressionInfo)

I think there is utils to read from map but not properties yet. We should refactor to unify the SBW and SBR code further. It is a future action item. Just mention it here.